PyInstaller/hooks/hook-django.py

```diff
@@ -76,23 +76,31 @@
     # Bundle django DB schema migration scripts as data files.
     # They are necessary for some commands.
     logger.info('Collecting Django migration scripts.')
-    django_mod_dir = os.path.dirname(get_module_file_attribute('django'))
     migration_modules = [
-             'conf.app_template.migrations',
-             'contrib.admin.migrations',
-             'contrib.auth.migrations',
-             'contrib.contenttypes.migrations',
-             'contrib.flatpages.migrations',
-             'contrib.redirects.migrations',
-             'contrib.sessions.migrations',
-             'contrib.sites.migrations',
+             'django.conf.app_template.migrations',
+             'django.contrib.admin.migrations',
+             'django.contrib.auth.migrations',
+             'django.contrib.contenttypes.migrations',
+             'django.contrib.flatpages.migrations',
+             'django.contrib.redirects.migrations',
+             'django.contrib.sessions.migrations',
+             'django.contrib.sites.migrations',
     ]
+    # Include migration scripts of Django-based apps too.
+    installed_apps = eval(get_module_attribute(package_name + '.settings', 'INSTALLED_APPS'))
+    for app in installed_apps:
+        migration_mod = app + '.migrations'
+        if migration_mod not in migration_modules:
+            migration_modules.append(migration_mod)
```

Comment:
You could us a `set()` here, like:
``
migration_modules.extend(set(app + '.migrations' for app in installed_apps))`
``